docs/en/week05/05-2.md

```diff
@@ -61,9 +61,7 @@ where $m_{t+1}$ is the momentum's exponential moving average.
 
 Bias correction that is used to keep the moving average unbiased during early iterations is not shown here.
 
-
```

Comment:
Why?

Reply:
Uhm that shouldn't have happened. Probably confirmed it in rebase instead of undoing it. Fixing w/ next commit.